windows/JackNetWinSocket.h

```diff
@@ -25,6 +25,8 @@
 #include <winsock2.h>
 #include <ws2tcpip.h>
 #include <stdint.h>
+#else
+#include <Winsock.h>
```

Comment:
Are you sure you can't use winsock2? It's supposed to replace winsock 1.

Reply:
halleluia, this could explain a runtime bug I'm getting ;) this was done in a too much of a hurry to clear the compile failures (not sure if it was Winsock2 or something else that causes dramatic redefinitions, but it should be fix-able)

Will change!